src/main/scala/io/github/gaelrenoux/tranzactio/doobie/package.scala

```diff
@@ -47,17 +44,20 @@ package object doobie extends Wrapper {
     private[tranzactio] override implicit val connectionTag: Tag[Connection] = doobie.connectionTag
 
     /** How to provide a Connection for the module, given a JDBC connection and some environment. */
-    final def connectionFromJdbc(env: Blocking, connection: JdbcConnection): ZIO[Any, Nothing, Connection] =
-      ZCatsBlocker.provide(env).map { b =>
-        val connect = (c: JdbcConnection) => Resource.pure[Task, JdbcConnection](c)
-        val interp = KleisliInterpreter[Task](b).ConnectionInterpreter
-        val tran = Transactor(connection, connect, interp, Strategy.void)
-        Has(tran)
-      }
+    final def connectionFromJdbc(env: TranzactioEnv, connection: JdbcConnection): ZIO[Any, Nothing, Connection] = {
+      ZIO.runtime[TranzactioEnv].flatMap { implicit r: zio.Runtime[TranzactioEnv] =>
+        ZIO.succeed[Connection] {
+          val connect = (c: JdbcConnection) => Resource.pure[Task, JdbcConnection](c)
+          val interp = KleisliInterpreter[Task].ConnectionInterpreter
+          val tran = Transactor(connection, connect, interp, Strategy.void)
+          Has(tran)
+        }
+      }.provide(env)
```

Comment:
@PawelJ-PL In your code, you used `provideLayer`, I think we can just use `provide` here.

Reply:
:+1: I can't remember, what I used `provideLayer` :smile: